src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_image.py

```diff
@@ -56,6 +56,7 @@ def test_read_images_from_alias_doc(self, mock_urlopen):
         self.assertEqual(parts[2], ubuntu_image['sku'])
         self.assertEqual(parts[3], ubuntu_image['version'])
 
+    @unittest.skip('It will not throw an exception now. Instead, it will failover to a local copy.')
```

Comment:
Why not change to test the new behavior ? And do we need to get it back in the future? If not, maybe we can just remove it.

Reply:
OK. I'll try to add one.

Reply:
Added